backend/business-logic/users.js

```diff
@@ -11,9 +11,8 @@ const userManager = {
         dob: userData.dob,
         gender: userData.gender,
         nationality: userData.nationality,
-        photo: userData.photo,
-        newcomer: userData.newcomer,
-        volunteer: userData.volunteer,
+        photo: userData.photoURL,
```

Comment:
we can use 
photoURL: userData.photoURL

Reply:
good catch, thanks!

Reply:
cool, can you update it and push?

Reply:
I could only make it in a new PR

Reply:
just push to 1-database-editSchema

Reply:
exactly, the branch `1-database-editSchema` is a protected one